recipes/pyjion/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.5" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tonybaloney/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 06a0a661ddfb884a341c6163a7ab7228be43a0ac43a29b397ce9e6e1d9e0f11e
+
+build:
+  number: 0
+  skip: True  # [py<310]
```

Comment:
@wolfv Looks like trying to use a selector for Python 3.10. Has `conda-build` actually resolved this? https://github.com/conda/conda-build/issues/4032